shared/health/src/main/java/io/pravega/shared/health/impl/HealthEndpointImpl.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.health.impl;
+
+import io.pravega.shared.health.ContributorNotFoundException;
+import io.pravega.shared.health.ContributorRegistry;
+import io.pravega.shared.health.Health;
+import io.pravega.shared.health.HealthContributor;
+import io.pravega.shared.health.HealthEndpoint;
+import io.pravega.shared.health.Status;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class HealthEndpointImpl implements HealthEndpoint {
+
+    private final ContributorRegistry registry;
+
+    HealthEndpointImpl(ContributorRegistry registry) {
+        this.registry = registry;
+    }
+
+    @NonNull
+    @Override
+    public Health getHealth(String id, boolean includeDetails) {
+        HealthContributor result = registry.get(id);
+        if (Objects.isNull(result)) {
```

Comment:
really, here you can just do `result == null`. I suggested `Objects.isNull` when you filter a `Stream`. But here you can just do this.